controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -208,6 +208,7 @@ void master_thread::handle_cmdu_1905_1_message(Socket *sd, ieee1905_1::CmduMessa
     }
     case ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WIFI_SIMPLE_CONFIGURATION_MESSAGE: {
         LOG(DEBUG) << "Received AP_AUTOCONFIGURATION_WIFI_SIMPLE_CONFIGURATION_MESSAGE";
+        handle_cmdu_1905_autoconfiguration_WSC(sd, cmdu_rx);
```

Comment:
If you changed to `bool handle_cmdu_1905_autoconfiguration_WSC` shouldn't you check the return value?

Reply:
This is for future validity checking which is irrelevant now for controller certificaiton.

Reply:
I'm with Semyon here.

Most of the other handle() functions return void because the caller doesn't actually handle errors. Anyway, if the handling "fails", it is not an error state that the caller could handle. You'll just not get an updated flow state.

If you do return something, it looks as if the caller is expected to handle something. It just makes the code harder to understand that way.

Also, at some point it would be good to enable the 'unused return value' warning. If a function returns something, it should be used.